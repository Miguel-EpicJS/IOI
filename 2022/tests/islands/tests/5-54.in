4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
8 9
1 6
5 3
0 5
0 2
4 3
6 1
2 1
7 4
3 7
