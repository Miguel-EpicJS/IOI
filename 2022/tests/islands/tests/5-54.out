91041670623a69783795ed82bda33426101ce311
OK
1
26
2 1 8 7 4 1 2 3 6 0 5 6 3 2 1 4 7 8 1 2 3 6 5 0 6 3
